test/init.js

```diff
@@ -3,15 +3,15 @@ module.exports = require('should');
 var DataSource = require('loopback-datasource-juggler').DataSource;
 
 var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
-
+console.log(config)
 global.getConfig = function (options) {
 
   var dbConf = {
-    host: config.host || 'localhost',
-    port: config.port || 3306,
+    host: process.env.MYSQL_HOST || config.host || 'localhost',
+    port: process.env.MYSQL_PORT || config.port || 3306,
     database: 'myapp_test',
-    username: config.username,
-    password: config.password,
+    username: process.env.MYSQL_USERNAME || config.username,
```

Comment:
Because that is what is being set by CI and used by other repos that look for it.